src/app/content/utils/getCleanContent.ts

```diff
@@ -24,4 +29,8 @@ export default function getCleanContent(
     // move (first-child) figure and table ids up to the parent div
     .replace(/(<div[^>]*)(>[^<]*<(?:figure|table)[^>]*?) (id=[^\s>]*)/g, '$1 $3$2 data-$3')
   ;
+
+  const fixRAPrelatives = (content: string) => fixRelativeURLs(replacements(content), ['a', 'img'], originalUrl);
+
+  return flow(fixRAPrelatives, transformer)(pageContent);
```

Comment:
as long as you're using flow you might as well put `replacements` on the beginning, its weird the way you have it